index.js

```diff
@@ -8,20 +8,22 @@ const refresh = require('./lib/refresh')
 
 const defaultApiUrl = process.env['MYTHX_API_URL'] || 'https://api.mythx.io'
 const defaultApiVersion = 'v1'
+const trialUserId = '123456789012345678901234'
 
 class Client {
   constructor (auth, inputApiUrl = defaultApiUrl) {
-    if (!auth) {
-      throw new TypeError('Please provide auth options.')
-    }
+    const { email, ethAddress, apiKey, password } = auth || {}
 
-    const { email, ethAddress, apiKey, password } = auth
+    let userId
+    if (!password && !email && !ethAddress && !apiKey) {
+      userId = trialUserId
+    }
 
-    if (!email && !ethAddress && !apiKey) {
+    if (password && !email && !ethAddress && !apiKey) {
       throw new TypeError('Please provide an user id auth option.')
     }
 
-    if (!apiKey && (!password && (email || ethAddress))) {
+    if (!apiKey && userId !== trialUserId && (!password && (email || ethAddress))) {
```

Comment:
I think we should remove `userId !== trialUserId` from the condition, with the current code it seems not possible given that we are setting it and it is not set from outside, WDYT? Maybe check for `!userId`?